docs/pages/inboxes/manage-inboxes.mdx

```diff
@@ -0,0 +1,189 @@
+```
+
+```kotlin [Kotlin]
+val inboxId = client.inboxIdFromAddress(address)
+```
+
+```swift [Swift]
+let inboxId = try await client.inboxIdFromAddress(address: address)
+```
+
+:::
+
+View the state of any inbox to see the addresses, installations, and other information associated with the `inboxId`.
+
+**Request**
+
+:::code-group
+
+```jsx [React Native]
+const state = await client.inboxState(true)
+const states = await client.inboxStates(true, [inboxId, inboxId])
+```
+
+```kotlin [Kotlin]
+val state = client.inboxState(true) 
+val states = client.inboxStatesForInboxIds(true, listOf(inboxID, inboxID))
+```
+
+```swift [Swift]
+let state = try await client.inboxState(refreshFromNetwork: true)
+let states = try await client.inboxStatesForInboxIds(
+	refreshFromNetwork: true,
+	inboxIds: [inboxID, inboxID]
+)
+```
+:::
+
+**Response**
+
+```json
+InboxState
+{
+	recoveryAddress: string,
+	addresses: string[],
+	installations: string[],
+	inboxId: string,
+}
+```
+
+## FAQ
+
+### What happens when a user removes a wallet address?
+
+- If a user has only one wallet address associated with their inbox:
+- If a user has multiple wallet addresses associated with their inbox:
+
+### How is the recovery address used?
+
+If a user ???, they can use their recovery address to ???
+
+To do this, provide a UX that enables a user to ???
+
+### If a user created two inboxes using two wallet addresses, is there a way to combine the inboxes?
+
+If a user logs in with wallet address 0x62EE...309c and creates inbox 1 and then logs in with wallet address 0xd0e4...DCe8 and creates inbox 2; there is no way to combine inbox 1 and 2.
+
+<div>
+  <img src="https://raw.githubusercontent.com/xmtp/docs-xmtp-org/0fa5d2dd3c00fb25c0adb7a615408574eb6864a7/docs/pages/img/two-inboxes.png" width="350px" />
+</div>
+
+You can add wallet address 0xd0e4...DCe8 to inbox 1, but both address 0x62EE...309c and 0xd0e4...DCe8 would then have access to inbox 1 only. Wallet address 0xd0e4...DCe8 would no longer be able to access inbox 2. 
+
+<div>
+  <img src="https://raw.githubusercontent.com/xmtp/docs-xmtp-org/0fa5d2dd3c00fb25c0adb7a615408574eb6864a7/docs/pages/img/two-inbox-remap-address.png" width="350px" />
+</div>
+
+To help users avoid this state, ensure that your UX surfaces their ability to add multiple wallet addresses to a single inbox.
+
+### What happens if I remove a wallet address from an inbox ID and then initiate a client with the private key of the removed address?
+
+**Does the client create a new inbox ID or does it match it with the original inbox ID the address was removed from?**
+
+The wallet address used to initiate a client should be matched to its original inbox ID. 
+
+You do have the ability to rotate inbox IDs if a user reaches the limit of 257 identity actions (adding, removing, or revoking addresses or installations). Hopefully, users won’t reach this limit, but if they do, inbox IDs have a nonce and can be created an indefinite number of times. 
```

Comment:
Should we say "indefinite" here - rather than "infinite", for example?

Reply:
Infinite might work better.